common/models.json

```diff
@@ -0,0 +1,17 @@
+{
+    "T1B1": {
+        "name": "Model One",
```

Comment:
Current device display name is e.g. "Trezor Model One", can you please add `Trezor ` prefix for all models?

Reply:
fixed in force-push